src/dbus/server/dbus_thread_object_rcp.cpp

```diff
@@ -1986,6 +2001,42 @@ otError DBusThreadObjectRcp::SetEphemeralKeyEnabled(DBusMessageIter &aIter)
     return error;
 }
 
+void DBusThreadObjectRcp::DeactivateEphemeralKeyModeHandler(DBusRequest &aRequest)
+{
+    otError error        = OT_ERROR_NONE;
+    auto    threadHelper = mHost.GetThreadHelper();
+
+    otBorderAgentClearEphemeralKey(threadHelper->GetInstance());
+    aRequest.ReplyOtResult(error);
+}
+
+void DBusThreadObjectRcp::ActivateEphemeralKeyModeHandler(DBusRequest &aRequest)
+{
+    otError     error        = OT_ERROR_NONE;
+    auto        threadHelper = mHost.GetThreadHelper();
+    uint32_t    lifetime     = 0;
+    uint16_t    port         = OTBR_CONFIG_BORDER_AGENT_MESHCOP_E_UDP_PORT;
```

Comment:
I think we don't need this variable. We can simply call it use the macro in the call.

Reply:
Accept. Thanks!